src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs

```diff
@@ -93,4 +94,11 @@ public class DbConfig : IDbConfig
     public bool EnableDbStatistics { get; set; } = false;
     public bool EnableMetricsUpdater { get; set; } = false;
     public uint StatsDumpPeriodSec { get; set; } = 600;
+
+    public int DisableAutoCompactions { get; set; } = 1;
+    public bool AdviseRandomOnOpen { get; set; } = true;
+    public bool AllowMmapReads { get; set; } = true;
```

Comment:
Probably should not set this to true by default also. https://github.com/facebook/rocksdb/issues/507

Reply:
Please do provide a direct citation of why?

Reply:
https://github.com/facebook/rocksdb/issues/507#issuecomment-74354158-permalink